modules/charon-core/src/main/java/org/wso2/charon3/core/attributes/MultiValuedAttribute.java

```diff
@@ -67,23 +69,23 @@ public Attribute getSubAttribute(String attributeName) throws CharonException {
      * @throws CharonException
      */
     @Override
-    public void deleteSubAttributes() throws CharonException {
+    public void deleteSubAttributes() {
```

Comment:
Changing public method signatures is not recommended. There may be users who use this method already. Changes should be done in a backward-compatible manner.